packages/apps-config/src/endpoints/productionRelayPolkadot.ts

```diff
@@ -204,21 +204,6 @@ export const prodParasPolkadot: Omit<EndpointOption, 'teleport'>[] = [
       logo: nodesDarwiniaSVG
     }
   },
-  {
-    homepage: 'https://efinity.io',
-    info: 'efinity',
-    paraId: 2021,
-    providers: {
-      Dwellir: 'wss://efinity-rpc.dwellir.com',
-      Efinity: 'wss://rpc.efinity.io',
-      OnFinality: 'wss://efinity.api.onfinality.io/public-ws'
```

Comment:
If incorrect for the parachain, and correct for the stand-alone chain, these providers should be moved to the production entry.

Reply:
But they took a recent decision to fork and start their own relay-chain and move the parachain to that chain.
https://twitter.com/enjin/status/1666818288616370177?t=6Hxo2-rutr_uKhK5buTRpw&s=19

Reply:
Cool. I just read a bit about that this morning.

All good - however it doesn't change the fact that they do have a parachain on Polkadot. So chain 2021 is still Effinity, although -

- from their side it is not meant to be used
- we really don't support forked relays at all

So in this case, the best course of action is just to remove the providers. This means -

1. In the parachain listing paraId 2021 is still named Effinity
2. There is no avenue to actually connect to the specific chain (due to the 2 points above)

TL;DR Only remove the ability to connect.